interfaces/repo_test.go

```diff
@@ -317,16 +343,23 @@ func (s *RepositorySuite) TestPlug(c *C) {
 }
 
 func (s *RepositorySuite) TestPlugSearch(c *C) {
-	addPlugsSlots(c, s.testRepo, `
+	addPlugsSlotsFromInstances(c, s.testRepo, "x", `
```

Comment:
I find this hard to read. Maybe:
```
type instanceNameAndYaml struct {
	Name string
	Yaml string
}

func addPlugsSlotsFromInstances(c *C, repo *Repository, iys []instanceNameAndYaml) []*snap.Info {
...
```
The full diff: http://paste.ubuntu.com/p/RBG9C3MGws/

Reply:
Thank you!. Applied the diff with some minor tweaks.